src/killall.ts

```diff
@@ -0,0 +1,148 @@
+const signals = [
```

Comment:
We should start adding comments at the top of the spec if they are or not Linux compatible: 
```suggestion
// Linux incompatible
const signals = [
```